share/src/main/java/io/zensoft/share/service/jobkg/auth/DefaultJobKgAuthorizationService.java

```diff
@@ -26,6 +28,7 @@ public DefaultJobKgAuthorizationService(JobKgUserProperties jobKgUserProperties)
     @Override
     public String authorize(JobKgUserProperties user) throws AuthorizationFailedException {
         try {
+            log.info("authorize method called");
```

Comment:
how about to indicate class to which the method `authorize()` belongs

Reply:
@aziret26 done